src/components/Navbar/index.tsx

```diff
@@ -8,5 +8,5 @@ export default function Navbar() {
 
   if (isMobile) return <MobileCollapse />
 
-  return <TopLeftNavItems desktop={desktop} />
+  return <DesktopNavigation desktop={desktop} />
```

Comment:
kinda weird to have an isMobile if check and a desktop prop

Reply:
Yeah, it's not wonderful, but just an in-between state until these checks are able to get refactored up stack.